Formula/ammonite-repl.rb

```diff
@@ -1,20 +1,20 @@
 class AmmoniteRepl < Formula
   desc "Ammonite is a cleanroom re-implementation of the Scala REPL"
   homepage "https://lihaoyi.github.io/Ammonite/#Ammonite-REPL"
-  url "https://github.com/lihaoyi/Ammonite/releases/download/0.6.2/ammonite-repl-0.6.2-2.11.8", :using => :nounzip
-  version "0.6.2"
-  sha256 "bd023f4ef27fe318fc5b0d0bf348895b20eb64067b828da034defc28aa98272d"
+  url "https://github.com/lihaoyi/Ammonite/releases/download/0.7.0/0.7.0", :using => :nounzip
+  sha256 "bcd86bee82b63f11bd4ae8b3c72b90401f3183b09f586326aeb6743f4c78f918"
 
   bottle :unneeded
 
   depends_on :java => "1.7+"
 
   def install
-    bin.install "ammonite-repl-0.6.2-2.11.8" => "amm"
+    bin.install Dir["*"].pop => "amm"
```

Comment:
`shift` probably makes sense then given the filename, but it seems likely either will keep working indefinitely unless upstream redesign the way they handle releases, and even then as you note the test will blow.